apps/backend/src/mcp/tools/device_info.py

```
@@ -9,9 +9,8 @@
 import logging
 import json
 import re
-from datetime import datetime
-from typing import Dict, List, Optional, Any
-from sqlalchemy.ext.asyncio import AsyncSession
+from datetime import datetime, timezone
+from typing import Any, Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)